algorithmic_efficiency/workloads/wmt/workload.py

```diff
@@ -95,6 +95,8 @@ def _build_input_queue(self,
       # Without the '+1' only `num_eval_train_examples-1` examples are used
       # since one example is filtered out in the input pipeline.
       split = f'train[:{self.num_eval_train_examples+1}]'
+    if split == 'test':
+      split = f'test[:{self.num_test_examples}]'
```

Comment:
See my comment above.

Reply:
Sorry, will revert this

Reply:
No worries, I already reverted this in #246.